Assets/LoggingManager/LoggingManager.cs

```diff
@@ -74,45 +61,27 @@ void Awake()
         }
     }
 
-    public void GenerateUIDs()
+    public void SetSavePath(string path)
     {
-        sessionID = Md5Sum(System.DateTime.Now.ToString(SystemInfo.deviceUniqueIdentifier + "yyyy:MM:dd:HH:mm:ss.ffff").Replace(" ", "").Replace("/", "").Replace(":", ""));
-        deviceID = SystemInfo.deviceUniqueIdentifier;
+        this.savePath = path;
     }
```

Comment:
shouldn't `SetSavePath()` also update the save paths in the `LogStore` (logsList) ?

Reply:
Actually the path is given to WriteToCSV objects in their constructor, so the savePath from Logging Manager will be provided to them during their creation.
Those objects are created at the end of the logging process when the user wants to save the data in files.

```c#
    private void SaveToCSV(string label)
    {
        if (!enableCSVSave) return;
        if (logsList.TryGetValue(label, out LogStore logStore))
        {
            WriteToCSV writeToCsv = new WriteToCSV(logStore, savePath, filePrefix, fileExtension);
            writeToCsv.WriteAll();
        }
    }
```
We can see that the savePath is used here (and only here).
Currently, a WriteToCSV object is created for each LogStore object that we want to save in a file.

Reply:
makes sense, thanks 🌴